apache-rat-core/src/main/java/org/apache/rat/report/xml/XmlReportFactory.java

```diff
@@ -72,6 +79,25 @@ public static RatReport createStandardReport(IXmlWriter writer,
 
         final IDocumentAnalyser[] analysers = {analyser, policy};
         DocumentAnalyserMultiplexer analysisMultiplexer = new DocumentAnalyserMultiplexer(analysers);
-        return new ClaimReporterMultiplexer(configuration.isDryRun(), analysisMultiplexer, reporters);
+        return new ClaimReporterMultiplexer(writer, configuration.isDryRun(), analysisMultiplexer, reporters);
+    }
+
```

Comment:
LHF: Pls reformat this block.

Reply:
Sry, Low Hanging Fruit (LHF) - I use this abbreviation in commits or reviews in case of minor details or simple things.